app/src/main/java/org/liberty/android/fantastischmemo/ui/StatisticsScreen.java

```diff
@@ -246,6 +257,8 @@ public Chart generateChart(BarData data) {
             chart.getXAxis().setTextColor(Color.WHITE);
             chart.getAxisLeft().setTextColor(Color.WHITE);
             chart.getAxisRight().setTextColor(Color.WHITE);
+            chart.getAxisLeft().setValueFormatter(new ChartYAxisValueFormatter());
```

Comment:
It won't couple main and inner class because you are using static inner class as you did in the previous commit, which is a good thing :)